library/std/src/os/wasi/io/mod.rs

```diff
@@ -2,5 +2,5 @@
 
 #![unstable(feature = "wasi_ext", issue = "71213")]
```

Comment:
Don't you also need the module itself to be stable?

Reply:
Ah, you're right. Fixed now.